### ISSUE TITLE
lighttpd service started but no page is displayed "The connection has timed out"

### ISSUE THREAD

**Issue by RV999** [pi-hole/pi-hole#691]

**Comment by RV999** [comment]
pi@DNS1:~ $ service lighttpd status
● lighttpd.service - Lighttpd Daemon
   Loaded: loaded (/lib/systemd/system/lighttpd.service; enabled)
   Active: active (running) since Tue 2016-09-13 23:01:56 CDT; 17h ago
  Process: 1973 ExecStartPre=/usr/sbin/lighttpd -t -f /etc/lighttpd/lighttpd.conf (code=exited, status=0/SUCCESS)
 Main PID: 1983 (lighttpd)
   CGroup: /system.slice/lighttpd.service
           ├─1983 /usr/sbin/lighttpd -D -f /etc/lighttpd/lighttpd.conf
           ├─1994 /usr/bin/php-cgi
           ├─1995 /usr/bin/php-cgi
           ├─1996 /usr/bin/php-cgi
           ├─1997 /usr/bin/php-cgi
           └─1998 /usr/bin/php-cgi

**Comment by RV999** [comment]
never mind, resolved

**Comment by PromoFaux** [comment]
What was the resolution?

**Comment by RV999** [comment]
Realization of my own stupidity. I was using 192.168.1.xx (copy-paste from documentation) instead of 192.168.0.xx. 
It was working all along.
I was suspicious of it due to the first install attempt where it didn't install lighttpd and php5 on a new install of a new RPi. I spent a few hours trying to install those two packages.

After reformatting the SD card reinstalling it, every thing installed but I still couldn't get to the admin page (my stupidity).
After opening this incident I realized i was using the wrong ip address.

The only difference between the two installs is the first time I set a static ip address before installing pi-hole.
